### PR TITLE
update field length

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

File: column/column.go [hunk 1]
```diff
@@ -167,6 +167,15 @@ func (c *Col) GetTypeDesc() string {
 			elems[i] = strings.Replace(c.Elems[i], "'", "''", -1)
 		}
 		buf.WriteString(fmt.Sprintf("('%s')", strings.Join(elems, "','")))
+	case mysql.TypeFloat, mysql.TypeDouble:
+		// if only float(M), we will use float. The same for double.
+		if c.Flen != -1 && c.Decimal != -1 {
+			buf.WriteString(fmt.Sprintf("(%d,%d)", c.Flen, c.Decimal))
+		}
+	case mysql.TypeTimestamp, mysql.TypeDatetime, mysql.TypeDate:
+		if c.Decimal != -1 && c.Decimal != 0 {
```
Comment (qiuyesuifeng) [inline column/column.go]: `c.Flen != -1`?

Comment (siddontang) [inline column/column.go]: time type only has decimal, flen is always -1

File: column/column.go [hunk 2]
```diff
@@ -167,6 +167,15 @@ func (c *Col) GetTypeDesc() string {
 			elems[i] = strings.Replace(c.Elems[i], "'", "''", -1)
 		}
 		buf.WriteString(fmt.Sprintf("('%s')", strings.Join(elems, "','")))
+	case mysql.TypeFloat, mysql.TypeDouble:
```
Comment (qiuyesuifeng) [inline column/column.go]: Seems `case mysql.TypeFloat, mysql.TypeDouble` is the same as default case, so is it needless?

Comment (siddontang) [inline column/column.go]: yes, float(8) will show float, not float(8) as the default

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**